client/src/modules/shopping-list/index.jsx

```diff
@@ -53,12 +53,36 @@ class ShoppingList extends Component {
     fetchItemsFromGivenList(id);
   };
 
+  handleClick = event => {
```

Comment:
we have a mess with handlers names... here is handleClick, and escapeHandler (below). In toolbar.jsx is clickListener and escapeListener...